pkgs/development/python-modules/linien-common/default.nix

```diff
@@ -0,0 +1,47 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, setuptools
+, importlib-metadata
+, numpy
+, rpyc
+, scipy
+, callPackage
+}:
+
+buildPythonPackage rec {
+  pname = "linien-common";
+  version = "0.8.0";
```

Comment:
Could you please update to version v1.0.0 ?

https://github.com/linien-org/linien/releases

Reply:
The issue you showed is fixed in many places in Nixpkgs with:

```nix
preCheck = ''
  export HOME=$(mktemp -d)
'';
```

Reply:
Issue still occurs after adding the `preCheck`.